src/neural/dx/network_dx.cc

```diff
@@ -24,6 +24,8 @@
   terms of the respective license agreement, the licensors of this
   Program grant you additional permission to convey the resulting work.
 */
+#include "network_dx.h"
```

Comment:
This one breaks the appveyor build

Reply:
Seems this is more complicated... Better move it back for now.

Reply:
I cannot, my editor formats files on save. :-P